src/cljs/athens/views/blocks.cljs

```diff
@@ -45,7 +46,22 @@
                                                      :top "2em"
                                                      :bottom "0"
                                                      :transform "translateX(50%)"
-                                                     :background (color :border-color)}]]})
+                                                     :background (color :border-color)}]
+                     [:&:after {:content "''"
```

Comment:
I've seen this pattern a few times `:before` or :`after` and then `:content "''"`. What does it do?

Is it being used the same way on lines 42, 50, and 63?

Reply:
@shanberg